photologue/models.py

```diff
@@ -51,6 +51,14 @@
 # Photologue image path relative to media root
 PHOTOLOGUE_DIR = getattr(settings, 'PHOTOLOGUE_DIR', 'photologue')
 
+# Use celery to speed up page loading after uploading photos?
+USE_CELERY = getattr(settings, 'PHOTOLOGUE_USE_CELERY', False)
+
+# When using celery, where do we want to temporarily store our zip-file after upload?
+TEMP_ZIP_STORAGE = getattr(settings, 'PHOTOLOGUE_TEMP_ZIP_STORAGE', FileSystemStorage())
```

Comment:
Use of FileSystemStorage will break the S3 compatibility. Well I think it's not working very well at the moment, but this will make it even more broken. We need a way to abstract away which file storage backend is used.

Reply:
I actually use FileSystemStorage() on AWS with elastic beanstalk.
When a developer wants to use S3 instead for zip-file uploads (for example when he wants to allow uploads of very large files, or he uses lambda or something similar) he can just set TEMP_ZIP_STORAGE to his default storage. I think this would need to be made clear in the documentation though.
I decided to set it to FileSystemStorage by default, because the file doesn't need to exist very long and you need quick access to it. After thinking about it though, it might make more sense to set its default to default_storage, since you'd otherwise get problems in environments where the worker hasn't got access to the same filesystem as the webserver.

I also think S3-compatibility was fixed by a PR 2 weeks ago. (#211)
I haven't tested it, but this is the exact fix I used to get it working on S3 on my site (The fix was already mentioned in the comments of some old issue in this repo)